kasa/cli/device.py

```diff
@@ -41,8 +41,14 @@ async def state(ctx, dev: Device):
     echo(f"Device state: {dev.is_on}")
 
     echo(f"Time:         {dev.time} (tz: {dev.timezone})")
-    echo(f"Hardware:     {dev.hw_info['hw_ver']}")
-    echo(f"Software:     {dev.hw_info['sw_ver']}")
+    echo(
+        f"Hardware:     {dev._device_info.hardware_version}"
```

Comment:
Maybe we should just add a separate print out for "region" and keep these only for versions?
Or should we even convert these to regular features?

Reply:
I feel like the region and the hardware version is quite tied together. For example when you look for firmware on the tplink website you only see certain hardware versions depending on which region's site you are on:

`IMPORTANT:  Model and hardware version availability varies by region. Please refer to your TP-Link regional website to determine`

So it seems logical to group them and keep the output more concise no?

Reply:
Yes, they are at times tightly coupled, probably partially due to different voltage, so the output format is fine. I'm still wondering if we should have a `model_region` property, for exactly the same reason.